ModulesSeparated/.AL-Go/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is it possible to move this AL-GO project under Modules/System?

Reply:
So the projects will be:
- DevTools
- System
- ModulesSeparates

I like that.

We can even re-brand ModulesSeparated to simply Modules. It's prettier, isn't it?

Reply:
The projects now are:

- `Modules` (all the system modules)
- `Modules/DevTools`
- `Modules/System` (System Application, it's tests and test libs)